pkg/vdbgen/vdb.go

```diff
@@ -475,6 +551,37 @@ func (d *DBGenerator) setSubclusterDetail(ctx context.Context) error {
 	return nil
 }
 
+// setImage will fetch the server version and use it to build
+// and set the image inside v.vdb.
+func (d *DBGenerator) setImage(ctx context.Context) error {
+	q := Queries[VersionQueryKey]
+	rows, err := d.Conn.QueryContext(ctx, q)
+	if err != nil {
+		return fmt.Errorf("failed running '%s': %w", q, err)
+	}
+	defer rows.Close()
+
+	if rows.Err() != nil {
+		return fmt.Errorf("failed running '%s': %w", q, rows.Err())
+	}
+	if !rows.Next() {
+		return errors.New("could not get Vertica version from meta-function")
+	}
+	var image string
```

Comment:
Suggest renaming this variable to `fullVersion`.  It's not the image name yet.  Its the output of the `version()` meta-function.

Reply:
Yes i don't know why I named it `image`, `fullVersion` is better.